en/docs/get-started/about-this-release.md

```diff
@@ -95,6 +95,12 @@ It is now available to download from [here](https://github.com/wso2/micro-integr
 - **[Generating service catalog entry (open API definition)]({{base_path}}/integrate/develop/working-with-service-catalog/)**
     WSO2 API Manager includes a Service Catalog where developers can register their services in a RESTful manner. Integration services can be created when deploying your project as a C-App. These integration services are made discoverable to the API Management layer via the Service Catalog so that API proxies can directly be created using them.
 
+#### Compare this release with previous versions of WSO2 EI 7.x
+
+Given below is a comparison of the Micro Integrator of API-M 4.0.0 and the previous WSO2 EI runtimes, particularly EI 7.x. Note that the Micro Integrator of API-M 4.0.0 is the latest, most improved version of the WSO2 EI runtimes. The Micro Integrator is one of the components of WSO2 EI and is now shipped with WSO2 API Manager 4.0.0.
```

Comment:
```suggestion
The Micro Integrator is the integration runtime of WSO2 API-M 4.0.0 as well WSO2 EI 7.x versions. The following changes are introduced with the Micro Integrator of API-M 4.0.0:

- The CLI tool of EI 7.x is replaced with the [API Controller (CLI)]({{base_path}}/install-and-setup/setup/api-controller/managing-integrations/managing-integrations-with-ctl) of API-M 4.0.0.
- The EI Kubernetes Operator of EI 7.x is merged with the [API Operator]({{base_path}}/install-and-setup/setup/kubernetes-operators/k8s-api-operator/overview) of API-M 4.0.0.

```

Reply:
We mention this already in the CLI and K8s sections. Do we need to repeat?

Also, it's slightly misleading because there are more changes than just those two.

Reply:
Yes, I understand what you mean by the second point Sam. I added a new line to my commit suggestion to address that. "Basically, in addition to the new features listed above, note the following changes".

I thought it is important to mention the CLI and K8s changes affecting the MI in this section because MI uses will always refer this section to understand what has changed in the MI. There is no other way for this user group to know that these changes were happened.